public/js/components/Autocomplete.js

```diff
@@ -51,7 +73,11 @@ const Autocomplete = React.createClass({
       dom.input(
         {
           ref: "searchInput",
-          onChange: (e) => this.setState({ inputValue: e.target.value })
+          onChange: (e) => this.setState({
+            inputValue: e.target.value,
+            selectedIndex: -1
+          }),
```

Comment:
This is good for now, but I wonder if we want to keep the selected state through searches